test/browser_tests/conf.js

```diff
@@ -203,17 +205,20 @@ var config = {
       windowHeightPx = Number( windowSizeArray[1] );
     }
 
-    browser.driver.manage().window().setSize(
-      windowWidthPx,
-      windowHeightPx
-    );
-
-    // Set default windowSize parameter equal to the value in settings.js.
-    browser.params.windowWidth = windowWidthPx;
-    browser.params.windowHeight = windowHeightPx;
-    browser.params.windowSize = String( windowWidthPx ) +
-                                ',' + String( windowHeightPx );
-
+    // Calling setSize with headless chromeDriver doesn't work properly if
+    // the requested size is larger than the available screen size.
+    if ( !envvars.HEADLESS_CHROME_BINARY ) {
+      browser.driver.manage().window().setSize(
+        windowWidthPx,
+        windowHeightPx
+      );
+
+      // Set default windowSize parameter equal to the value in settings.js.
+      browser.params.windowWidth = windowWidthPx;
+      browser.params.windowHeight = windowHeightPx;
+      browser.params.windowSize = String( windowWidthPx ) +
```

Comment:
I don't think you need to use `String` here. I would try  = ${windowWidthPx},{windowHeightPx}. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

or just `windowWidthPx + ',' + windowHeight`